src/main/resources/avro/referencemethods.avdl

```diff
@@ -169,6 +169,11 @@ The query parameters for a request to `GET /references/{id}/bases`, for
 example:
 
 `GET /references/{id}/bases?start=100&end=200`
+
+_Error Conditions_
+
+- If `start` or `end` exceed the index of the last base of the `Reference`, this method will throw `GAException` and set the HTTP
```

Comment:
how about `out of range of the reference?

Jerome Kelleher <notifications@github.com> writes:
> In src/main/resources/avro/referencemethods.avdl:
> 
> > @@ -169,6 +169,11 @@ The query parameters for a request to `GET /references/{id}/bases`, for
> >  example:
> >
> >  `GET /references/{id}/bases?start=100&end=200`
> > +
> > +_Error Conditions_
> > +
> > +- If `start` or `end` exceed the index of the last base of the `Reference`, this method will throw `GAException` and set the HTTP
> 
> I don't think this quite captures it. What about
> 
>     If start or end are negative, or greater than the length of the reference,
>     this method ....
> 
> —
> Reply to this email directly or view it on GitHub.*
>